package.json

```diff
@@ -11,6 +11,6 @@
     "@playwright/test": "^1.41.1",
     "@types/node": "^20.11.11",
     "prettier": "^2.7.1",
-    "turbo": "^1.4.3"
+    "turbo": "^1.13.3"
```

Comment:
Update to the latest stable version of `turbo` so that it can be run from anywhere in the repo, see https://github.com/vercel/turbo/issues/1412